components/vars/css/components/spectrum-actionbutton.css

```diff
@@ -122,130 +122,136 @@
   --spectrum-actionbutton-s-quiet-emphasized-texticon-text-letter-spacing: var(--spectrum-global-font-letter-spacing-none);
   --spectrum-actionbutton-s-quiet-emphasized-texticon-text-text-transform: none;
   --spectrum-actionbutton-s-quiet-emphasized-texticon-text-font-size: var(--spectrum-global-dimension-font-size-75);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-padding-top: var(--spectrum-global-dimension-size-50);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-padding-bottom: var(--spectrum-global-dimension-size-50);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-margin-top: 0;
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-margin-bottom: 0;
   --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-render: yes;
   --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-order: 1;
   --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-size: var(--spectrum-global-dimension-size-200);
-  --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-padding-top: var(--spectrum-global-dimension-size-50);
-  --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-padding-bottom: var(--spectrum-global-dimension-size-50);
   --spectrum-actionbutton-s-quiet-emphasized-texticon-icon-gap: var(--spectrum-global-dimension-size-85);
-  --spectrum-actionbutton-s-quiet-emphasized-texticon-focus-ring-border-radius: var(--spectrum-global-dimension-size-100);
   --spectrum-actionbutton-s-quiet-emphasized-texticon-focus-ring-size: var(--spectrum-alias-focus-ring-size);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-focus-ring-border-width: var(--spectrum-alias-focus-ring-size);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-focus-ring-border-radius: var(--spectrum-alias-border-radius-medium);
   --spectrum-actionbutton-s-quiet-emphasized-texticon-layout: row;
   --spectrum-actionbutton-s-quiet-emphasized-texticon-min-width: var(--spectrum-global-dimension-size-400);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-gap: var(--spectrum-global-dimension-size-85);
   --spectrum-actionbutton-s-quiet-emphasized-texticon-height: var(--spectrum-global-dimension-size-300);
-  --spectrum-actionbutton-s-quiet-emphasized-texticon-border-radius: var(--spectrum-global-dimension-size-50);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-border-radius: var(--spectrum-alias-border-radius-regular);
+  --spectrum-actionbutton-s-quiet-emphasized-texticon-#padding-text: var(--spectrum-global-dimension-size-115);
```

Comment:
We're getting `#` symbols in our variables names.

Reply:
Looks like it's a fairly widespread problem.

* actionbutton
* button
* checkbox
* clearbutton
* cyclebutton
* link
* listitem
* panel
* radio
* rating
* scrollbar
* search
* switch
* tag
* textarea
* textfield

Reply:
Looks like it's coming from `@adobe/spectrum-tokens`. I double checked and this is on version `7.0.0-beta.21`